src/internal/converters/eml/eml_test.go

```diff
@@ -188,3 +191,137 @@ func (suite *EMLUnitSuite) TestConvert_edge_cases() {
 		})
 	}
 }
+
+func (suite *EMLUnitSuite) TestConvert_eml_ics() {
```

Comment:
both of the input data types in the test seem like they're for the `EventMessageRequestable` interface. Could you please make some basic tests for the other types so we have something to ensure we don't lose that functionality going forward?

Reply:
We don't handle the others as the data is incomplete. Let me update the code to reflect that.